pexpect/expect.py

```diff
@@ -263,25 +251,16 @@ def __str__(self):
         ss = list(zip(*ss))[1]
         return '\n'.join(ss)
 
-    def search(self, buffer, freshlen, searchwindowsize=None):
```

Comment:
I'm not sure that we can get rid of these parameters so easily - backwards compatibility again.